### ISSUE TITLE
AL Language Extension 12.x Preview false positive LC0003 LC0005

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#297]
I installed the preview version of the AL Language Extension and
I receive the following warnings:
warning LC0003: Do not use an Object ID for properties or variables declaration. Use tabledata "BOM Component" = R instead.
warning LC0005: Wrong casing detected! Use tabledata "BOM Component" = R instead.

```al
action("PTE Action)
{
                AccessByPermission = tabledata "BOM Component" = R;
                ApplicationArea = All;
                Caption = 'Assembly Order';
                Image = AssemblyOrder;
                ShortcutKey = 'Ctrl+M';
                ToolTip = 'View any linked assembly.';

                trigger OnAction()
                var
                    ATOLink: Record "Assemble-to-Order Link";
                begin
                    ATOLink.ShowAsm(Rec);
                end;
}
```

**Comment by pri-kise** [comment]
I had to reload VS Code once again and now this seems to be solved. At least partially.

I still receive a similiar error here inside a table.
```al
field(19; Date; Date)
        {
            AccessByPermission = tabledata "Sales Shipment Header" = R;
            Caption = 'Order Date';
        }
```
Or do I have change anything else there?
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/56249171/ed01e217-55be-433f-910a-c8357587ff5f)

**Comment by mmenninghaus** [comment]
We are experiencing the same issue. Any update on this?

**Comment by Arthurvdv** [comment]
With ALLanguage v13.0.887907 and the prerelease version of the LinterCop, I can not reproduce this issue.

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/44637996/08b92ce6-3b95-4129-a5f9-0a83c0279760)

The example file below shows no LC003 or LC005 warnings.

```AL
table 50100 "MyTable"
{
    fields
    {
        field(19; "Date"; Date)
        {
            AccessByPermission = tabledata "Sales Shipment Header" = R;
            Caption = 'Order Date';
        }
    }
}

**Comment by pri-kise** [comment]
This issues seems to be gone.
